CHANGELOG.md

```diff
@@ -124,6 +143,24 @@ _Note: this is a manually generated changelog for initial v2 Alpha release due t
 - Skeletal gRPC connection for "getAllRows()" (#1363)
 - First skeletal version of SGv2 rest-service (#1357)
 
+## [v1.0.51](https://github.com/stargate/stargate/tree/v1.0.51) (2022-03-14)
+
+[Full Changelog](https://github.com/stargate/stargate/compare/v2.0.0-ALPHA-4...v1.0.51)
```

Comment:
Should be 1.0.51 to 1.0.50?

Reply:
Likewise, yup. I should have paid closer attention here: I knew how these should look like.

Reply:
Created #1702 for SGv1 side where this came from.